src/main/java/org/mastodon/app/ui/ViewMenu.java

```diff
@@ -52,7 +52,14 @@
 
 public class ViewMenu
 {
-	private static final boolean USE_ACCELERATORS = MastodonDebugSettings.getInstance().isUseMenuAccelerators(); // TODO: remove, once Fiji ships at least jdk1.8.0_162
+	private static final boolean USE_ACCELERATORS = MastodonDebugSettings.getInstance().isUseMenuAccelerators(); // TODO:
+																													// remove,
+																													// once
+																													// Fiji
+																													// ships
+																													// at
+																													// least
+																													// jdk1.8.0_162
```

Comment:
The formatter should allow a comment at the end, and not break it automatically.